index.html

```diff
@@ -5,19 +5,49 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link rel="stylesheet" href="style.css">
+    <script src="./navigation.js" defer></script>
     <title>Awesome Books</title>
 </head>
 <body>
-    <h1 id="list-title">Awesome Books</h1> 
-    <div id="book-list">
-    <!--Booklist added in javascript  -->
-    </div>
-    <form id="book-form">
-        <h2 id="form-title">Add new book</h2>
-        <input type="text" id="title" placeholder="Title" required>
-        <input type="text" id="author" placeholder="Author" required>
-        <button type="submit">Add</button>
-    </form>
+    <header>
+        <nav>Awesome book
+            <ul id="navbar">
+                <li class="nav-btn" data-id="book-list-section">Book-list</li> 
+                <li class="nav-btn" data-id="form">Add new</a></li> 
+                <li class="nav-btn" data-id="Contact">Contact</li> 
+            </ul>
+        </nav>
+    </header>
+    <main>
+        <section id="book-list-section">
+            <h1 id="list-title">Awesome Books List</h1> 
+            <div id="book-list">
+            <!--Booklist added in javascript  -->
+            </div>
+        </section>
+        <section id="form">
+            <form id="book-form">
+                <h2 id="form-title">Add new book</h2>
+                <input type="text" id="title" placeholder="Title" required>
+                <input type="text" id="author" placeholder="Author" required>
+                <button type="submit">Add</button>
+            </form>
+        </section>
+
+        <section id="Contact">
+            <h3>Contact Information</h3>
+            <div>
+                <p>If you have Queries or suggestion reach us at:</p>
+                <ul>
+                    <li> Our support email: support@awesomebook.com</li>
+                    <li>You can call us: 0532214239</li>
+                    <li>Our address is: New Human Colony, Mars</li>
+                </ul>
+            </div>
+        </section>
+    </main>
+
+    <footer>Copyright@2023</footer>
 
```

Comment:
- [ ] Great work on implementing the requirements of this project 👍 but this project has a crucial requirement of pair programming 🤔 I cannot see any commit from one of your team members @bertinkambale 😨 Kindly ask him to contribute to your project to follow this crucial requirement 👍

Reply:
Thank you 👍🏾 for the review. The one **@bertinkambale** unexpectedly quit the coding partners group. We got a message from the Otto bot on the slack channel that Bertin will no longer be with us. 😞